awslabs/aws-lambda-powertools-typescript
# feat(logger): adopted Utility class & updated unit tests

## Description of your changes

This PR introduces changes to the `Logger` module to adopt the forthcoming `Utility` class that will be released in the `@aws-lambda-powertools/commons` package.

- [x] Install released `@aws-lambda-powertools/commons` package
- [x] Make changes to the code to use the `Utility` class
- [x] Run unit & e2e tests
- [x] Add note in the docs about initialising utility outside of function handler ([ref](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/25fd4a0c2f40ff5af5ee6ec08a40a08f3e2dc22d/packages/commons/src/Utility.ts#L26))

### How to verify this change

See results of PR checks, also see the screenshot below that shows a successful e2e test run (on local):
WIP

Optionally, re-run the e2e tests on GitHub Actions

### Related issues, RFCs

[#484](https://github.com/awslabs/aws-lambda-powertools-typescript/issues/484)  
[#547](https://github.com/awslabs/aws-lambda-powertools-typescript/pull/547)

### PR status

***Is this ready for review?:*** NO  
***Is it a breaking change?:*** NO

## Checklist


### Breaking change checklist

N/A

---

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
